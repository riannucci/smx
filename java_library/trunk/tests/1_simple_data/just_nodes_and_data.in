node >
   data for node
   i > Some data surrounded by i
   """Some string data for node"""
   d > 'Some string data surrounded by d'
