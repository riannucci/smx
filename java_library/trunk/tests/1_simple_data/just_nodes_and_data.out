<?xml version="1.0" encoding="UTF-8"?>
<node>data for node<i>Some data surrounded by i</i>Some string data for
        node<d>Some string data surrounded by d</d>
</node>
